### DISCUSSION TITLE
How to use snow cover and LST in mHM

### DISCUSSION THREAD

**Discussion by fmanquehual** [mhm-ufz/mHM#56] [Q&A]
Hi everyone!

I am using mHM to simulate the streamflow in a basin in the Chilean Patagonia.

The purpose of my message is the following. According to the mHM page (https://mhm.pages.ufz.de/mhm/stable/data.html) it is possible to use snow cover and land surface temperature as optional data, the issue is that reviewing the mhm.nml file, I can't find the way to "enable" its use in the model. Therefore, how can I enable it?

**Comment by Joaquinfnz** [comment]
i dont know grettings from chile
